scripts/gui_main.py

```diff
@@ -1,33 +1,119 @@
 import tkinter as tk
-
+from tkinter import *
+from tkinter import ttk
 
 from Fly_Swatter.Fly_Swatter import mode_control, fire_mode, search, radar, target, graph_trajectory
 
 class GUI_App:
     def __init__(self):
         
         self.root = tk.Tk()
-        self.var_0 = tk.BooleanVar()
-        self.var_1 = tk.BooleanVar()
-        self.var_2 = tk.IntVar(value = 1)
-        self.var_3 = tk.BooleanVar()
         
-        self.verbose_button = tk.Checkbutton(self.root, text='Verbose', variable = self.var_0, command = self.check_button).pack()
-        self.graph_button = tk.Checkbutton(self.root, text='Graphical', variable = self.var_1, command = self.check_button).pack()
-        self.realism_button = tk.Checkbutton(self.root, text='Realism', variable = self.var_2, command = self.check_button).pack()
-        self.engage_button = tk.Checkbutton(self.root, text='Engage', variable = self.var_3, command = self.check_button).pack()
+        self.frame = tk.Frame(self.root, relief = tk.GROOVE, borderwidth = 5)
```

Comment:
tkinter uses a set of int as variables for checkboxes while the program as it is currently written uses booleans; may just rewrite program to use ints